opennms-provision/opennms-provisiond/src/test/java/org/opennms/netmgt/provision/service/NodeLocationChangeIT.java

```diff
@@ -161,6 +168,9 @@ public void testNodeLocationChange() throws Exception {
         OnmsNode node = nodes.get(0);
         assertEquals("Hyderabad", node.getLocation().getLocationName());
         assertEquals(1, node.getIpInterfaces().size());
+        assertEquals(1, node.getSnmpInterfaces().size());
```

Comment:
This test validates changes in ScanManager where  hostName is actually getting overwritten